src/pages/StudentDashboardPage/DashboardComponents/DashboardHeader.js

```diff
@@ -0,0 +1,72 @@
+      />
+      <Box className={classes.nameContainer}>
+        <Typography variant="h5">
+          <Typography component="span" variant="h5" className={classes.spantext}>
+            Hi
+          </Typography>
+          Code For Cause
+        </Typography>
```

Comment:
Also add mobile Number, for now you can show emailId, since we don't have phone number.
![image](https://user-images.githubusercontent.com/48255244/113582360-e8592c00-9645-11eb-91b9-e68471b404db.png)

Reply:
Sure thing! @Abhishek-kumar09  I'll do the change :)